CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing to Freemocap
```

Comment:
These are the new contribution guidelines, please read and lets discuss anything thats weird, wrong, strange, or lets add whatever else we need to add.